pom.xml

```diff
@@ -82,6 +82,8 @@
             <phase>generate-sources</phase>
             <configuration>
               <outputDirectory>target/generated-sources/antlr4/org/apache/accumulo/access/grammars</outputDirectory>
```

Comment:
What's wrong with the default output directory at: `${project.build.directory}/generated-sources/antlr4`?

Reply:
IIRC I ended up with this because the output generated java classes with the correct package name, but put them in that directory. It didn't create the correct directory structure. I can re-test this to be sure. I think this issue points to the same problem: https://github.com/antlr/antlr4/issues/4202

Reply:
I noticed the directory structure issue when I ran it, but it didn't seem to matter for the build. Since the files aren't checked in from the target directory, it probably doesn't matter.